src/cmds/init/index.ts

```diff
@@ -16,9 +16,8 @@ module.exports.builder = (yarg: yargs.Argv) => {
 module.exports.handler = (argv: YargsArgs) => init(argv);
 
 async function init(argv: YargsArgs): Promise<void> {
-  const newArgs = await initHandler(argv);
 
-  console.log();
+  const newArgs = await initHandler(argv);
```

Comment:
I think the `console.log()` here was intentional to create a newline in the user's terminal.